Dockerfile-kubebuilder

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
```

Comment:
Can you rename this `Dockerfile`? No reason to have both. Unsure if we want this to be the kubebuilder generated makefile or the standard GS makefile.

Reply:
On second thought, using the normal kubebuilder makefile seems better so we match CAPA/CAPZ/CAPI.

Reply:
Yeah I was just purely concentrated on code for now, kept both `Make-kubebuilder` and `Dockerfile-kubebuilder` just so I don't lose them till I figure out how to merge them with the current ones.